demisto_sdk/commands/content_graph/strict_objects/base_strict_model.py

```diff
@@ -104,6 +104,12 @@ class StructureError(BaseStrictModel):
     error_type: Optional[str] = Field(None, alias="type")
     ctx: Optional[dict] = None
 
+    def __str__(self):
+        return (
+            f"problematic field: {self.field_name} | error message: {self.error_message} |"
+            f" error type : {self.error_type}"
```

Comment:
Let's write a nice message for the common (extra forbidden, missing, invalid type, can't be none), and leave some default for all the rest.

Reply:
Done